rbac/management/group/definer.py

```diff
@@ -57,6 +57,24 @@ def seed_group(tenant):
                 logger.info("Finished seeding default group %s for tenant %s.", name, tenant.schema_name)
             else:
                 logger.info("Default group %s is managed by tenant %s.", name, tenant.schema_name)
+
+            # Default admin group
+            admin_name = "Default Admin access"
```

Comment:
For case consistency, we should probably update this to be `Default admin access`
```suggestion
            admin_name = "Default admin access"
```

Reply:
It's now lowercase.

Reply:
Awesome, thanks!